docs/tutorial/security.md

```diff
@@ -48,21 +48,21 @@ the result of the overall security of the framework foundation
 your code. As such, it is your responsibility to follow a few important best
 practices:
 
-* **Keep your application up-to-date with the latest Electron framework release.** 
```

Comment:
even though I don't like trailing whitespace, we should not include this change

Reply:
Restored.